docs/tutorials/01-intro.md

```diff
@@ -107,7 +107,7 @@ By echoing `$scratchmnt` we can see the path for the [overlay image](https://wik
 
 Notice that the overlay image is under `/var/lib/containers/storage` as one would expect. (See above on `containers/storage` or for more information see [containers/storage](https://github.com/containers/storage).)
 
-Now that we have a new empty container we can install or remove software packages or simply copy content into that container. So let's install `bash` and `coreutils` so that we can run bash scripts. This could easily be `nginx` or other packages needed for your container.
+Now that we have a new empty container we can install or remove software packages or simply copy content into that container. So let's install `bash` and `coreutils` so that we can run bash scripts. This could easily be `nginx` or other packages needed for your container.  **NOTE:** the version in the example below (30) relates to a Fedora version which is the Linux platform this example was run on.  If you are using the scratch image, the version you specify must be valid for the host or dnf will throw an error.  I.e. If you were to run this on a RHEL platform, you'd need to specify `--releasever 8.1` or similar instead of `--releasever 30`.  If you want the container to be a particular Linux platform, change `scratch` in the first line of the example to the platform you want, i.e. `# newcontainer=$(buildah from fedora)`, and then you can specify an appropriate version number for that Linux platform.
```

Comment:
I would prefer the NOTE to be on a new line.  Could you update the "30" to something more recent, say "33" in all of the examples.

Reply:
I also thought you like to avoid using "you" in documentation?

Reply:
"you" is verboten in man pages, OK in tutorials and blog posts as long as it's not overused.  Will adjust this on Tuesday, now it's almost time for my favorite little man to arrive for a few hours.

Reply:
Can you move the NOTE to it's own line.?

Reply:
Done, just unburied the inbox, update inc.

Reply:
You also did not update the version from 30 to 33 so that it is a supported version of Fedora for the next 1.5 years.